base_location_geonames_import/wizard/geonames_import.py

```diff
@@ -25,13 +26,8 @@
 class CityZipGeonamesImport(models.TransientModel):
     _name = "city.zip.geonames.import"
     _description = "Import City Zips from Geonames"
-    _rec_name = "country_id"
 
-    country_id = fields.Many2one("res.country", "Country", required=True)
-    code_row_index = fields.Integer(
```

Comment:
These changes for example

Reply:
Hi! Thank you for your answer. I removed the related fields code_row_index and name_row_index because country_id field has been converted into a Many2many field (it was a Many2one field before the changes I applied). That is also why _rec_name has been removed.

Reply:
OK, understood. You're right.